EIPS/eip-4762.md

```diff
@@ -178,6 +203,9 @@ For a transaction, make these access events:
 (tx.target, 0, BASIC_DATA_LEAF_KEY)
 (tx.target, 0, CODEHASH_LEAF_KEY)
 ```
+These accesses are not charged gas costs (i.e: included in intrinsic gas).
+
+If `tx.target` is an EOFv1 account, access code chunks corresponding to bytecode from the entire `header` and `type_section[0]` of the container. These accesses are charged gas costs, and done immediately after the above accesses.
```

Comment:
This is a similar rule to the already mentioned instructions that jump to a new container. Also specified where this access happen (which I believe is something we need to do better in general in many parts of this EIP).